blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseBlobIntegrationTest.java

```diff
@@ -685,6 +686,32 @@ public void testPutMultipartInputStream() throws Exception {
       testPut(payload, null, new ByteSourcePayload(byteSource), length, new PutOptions().multipart(true));
    }
 
+   @Test(groups = { "integration", "live" })
+   public void testPutHugeObject() throws Exception {
```

Comment:
We cannot add this to the general integration test group since it will explode run times.  It looks like we have a "performance" group for this; could you use that tag instead?